File: app/src/main/java/com/example/olesya/boardgames/interfaces/ItemCallback.java
package com.example.olesya.boardgames.interfaces;

import com.example.olesya.boardgames.Entity.Card;

public interface ItemCallback {

    void onSelectedCardEvent(Card username);

    void onUserTurnFinished(Card card);

    void onAddUserChoice(String clientName, int currentChoice);
}

File: app/src/main/java/com/example/olesya/boardgames/interfaces/ItemCallback.kt
package com.example.olesya.boardgames.interfaces

import com.example.olesya.boardgames.entity.Card

interface ItemCallback {

    fun onSelectedCardEvent(username: Card)

    fun onUserTurnFinished(card: Card)

    fun onAddUserChoice(clientName: String, currentChoice: Int)
}